### ISSUE TITLE
[New build - DEFAULT] 24w07a: More Survival hotfixes

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Starship-EVO#5532]
Another hotfix build with mainly survival fixes.

Hotfixes:
#5463 Ship shake when hitting water on low FPS.
#5445 Ship cannot jump in lightcruise with low FPS
#5491 Player stuck in a wreck on survival start
#5503 Broken brick leave children brick on reload
#5487 Symmetry does not consume bricks
#5500 Starter block does not consume item quantity
#5493 Can pick tree and crystals
#5510 Gravity unit require wrong type of crystal
#5497 Copy-paste does not consume quantity

**Comment by tsunamayo** [comment]
> Does the light cruise jump correction also work for warp jump?

You tell me! I was not aware of problem for warp, the fix was on the way a ship auto-pilot aligns to its target, so if thats was the problem it should be fixed.

**Comment by Aranhanauta** [comment]
Yes, is fixed, thank you so much. I finay can explorate the galaxy

**Comment by Uncle-Ulty** [comment]
Nice! Gonna start a new survival mode and check for new bugs
